src/core/net/private/geoipdatabase.h

```diff
@@ -49,12 +52,48 @@ class GeoIPDatabase
     QString type() const;
     quint16 ipVersion() const;
     QDateTime buildEpoch() const;
-    QString lookup(const QHostAddress &hostAddr) const;
+    QString lookup(const QHostAddress &hostAddr);
```

Comment:
fromBigEndian() is const.
But that's not it. I understand that once you have removed const modifier means something prevented you to leave it.
Therefore, I will describe the situation in General. This method should be logically const. If something prevents this, it looks like an error in class design. Look closely, what do you have to change inside the method. If it is related only with countries caching, make it mutable.